### DISCUSSION TITLE
Mapping relations with SQLite [Feb 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5110] [Discord Threads]
> 🙋‍♂️ **sangie**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sangie**  
#### I hope there is something stupid I'm missing.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061657510534205490)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
